homeworks/Aleksandr.Turuta_turuta7/homework_6/1.js

```diff
@@ -0,0 +1,100 @@
+// the task 1
+const user = {};
+Object.defineProperties(user, {
+    fullName: {
+        enumerable: true,
+        configurable: true,
+        writeble: true,
+        set: (name) => {
+            this.fullName = name
+                .replace(/([[A-Za-z])\S/g, s => s.toLowerCase())
+                .replace(/(^|\s|[-])\S/g, s => s.toUpperCase());
+        },
```

Comment:
looks a bit complicated... you could check for every first letter in a word `\b\w` for example